upgrading/change-log.md

```diff
@@ -1,35 +1,51 @@
-- FusionCharts version 3.20 updated `Angular` integration to support Angular versions 14 and 15.
-- FusionCharts version 3.20 updated `React Native` plugin to support React Native versions 0.68 and 0.69.
-- FusionCharts version 3.20 updated the wrappers for `Python`, `C#`, and `Java SDK` to the latest supported versions. 
-- FusionCharts version 3.20 added the new property `orientedTextAlign`, which allows users to align the axis text.
+- FusionCharts version 3.21 updated the `Ramda` dependency package version to the latest version, version 0.29. 
+- FusionCharts version 3.21 introduces the new function that removes any cross-site scripting that might be causing errors in your code.
+- FusionCharts version 3.21 introduces new maps for all 24 Argentinian departments, all 26 Peruvian departments, and all 7 provinces from Costa Rica. 
 
 
 <h4>Improvements</h4>
 
-- FusionCharts version 3.20 improved the chart’s resizing functionality for gauges. By adding an or condition to the `colorRange` method, resizing the chart container adjusts the chart’s width accordingly.
-- FusionCharts version 3.20 improved the Radial Bar chart by adding support to mouse click events such as `dataPlotClick`, and `dataPlotRollOver` and includes the new `plotEventHandler` method to bind the mouse events with the chart events.    
-- FusionCharts version 3.20 improved the Time Navigator time format to match the x-axis labels' time format.
-- FusionCharts version 3.20 includes the possibility to configure the width for the legend container with the new `disableLegendScroll` property, which, when set to 1, disables the scrollbar giving the legend container the necessary space. 
+- When using Multiseries Line charts, similar values overlapped in the chart. FusionCharts version 3.21 improved the `drawcrossline` attribute to prevent overlapping values and display a clean chart.  
+- Improved the `numberSuffix` attribute to work as expected when using numeric and Arabic characters. Previously, the characters were displayed in an incorrect order. Now, enabling the `hasRtlText` attribute corrects the suffix and prefix positions, showing the characters in the correct order. 
+- Improved the `Sunburst` chart by adding the `showLabelValue` property that allows users to control label visibility for each node at all levels in the chart. 
+- Improved the `dataLabelstyles` function to handle the properties, attributes, and conditions for the data label styles. Previously, when `plotHighlightEffect` was set, and if value-related cosmetics were applied at the dataset level, the value-related cosmetics did not display as expected upon hovering on the legends.
+- Improved the accessibility feature. Now, when using React or Angular, you can enable the accessibility feature to import directly from the node modules packages.
```

Comment:
This feature already existed, the reason it was raised as a bug was that the user did not enable 'accessibility' attribute in the chart configuration. I think this needs to be rephrased, please check comment discussion in the ticket: https://fusioncharts.jira.com/browse/SUPPORT-2285